packages/generator/__tests__/generate-recipe.test.ts

```diff
@@ -78,24 +78,20 @@ describe('generate recipes', () => {
 
       const textStyleFn = createRecipe('textStyle', {}, [])
 
-      const variantKeys = [
-        \\"size\\"
-      ]
-
-      function splitVariantProps(props) {
-        return splitProps(props, variantKeys)
-      }
-
       export const textStyle = Object.assign(textStyleFn, {
         __recipe__: true,
-        variantKeys,
+        variantKeys: [
+        \\"size\\"
+      ],
         variantMap: {
         \\"size\\": [
           \\"h1\\",
           \\"h2\\"
         ]
       },
-        splitVariantProps,
+        splitVariantProps(props) {
+          return splitProps(props, variantKeys)
```

Comment:
There's no reference to variantKeys here and this line will throw an error.

You can consider inlining the code for `variantKeys` there as well